migrations/m160509_211405_add_flashed_to_notification.php

```diff
@@ -6,11 +6,11 @@ class m160509_211405_add_flashed_to_notification extends Migration
 {
     public function up()
     {
-        $this->addColumn('notification', 'flashed', $this->boolean()->notNull());
+        $this->addColumn('{{%notification}}', 'flashed', $this->boolean()->notNull());
```

Comment:
this table name is more general, independent on database type (quoting) and table prefix
[http://www.yiiframework.com/doc-2.0/guide-db-dao.html#quoting-table-and-column-names](http://www.yiiframework.com/doc-2.0/guide-db-dao.html#quoting-table-and-column-names)